src/app/clusters/time-synchronization-server/DefaultTimeSyncDelegate.h

```diff
@@ -36,6 +36,7 @@ class DefaultTimeSyncDelegate : public Delegate
     CHIP_ERROR UpdateTimeFromPlatformSource(chip::Callback::Callback<OnTimeSyncCompletion> * callback) override;
     CHIP_ERROR UpdateTimeUsingNTPFallback(const CharSpan & fallbackNTP,
                                           chip::Callback::Callback<OnFallbackNTPCompletion> * callback) override;
+    void UTCTimeChanged(uint64_t time) override;
```

Comment:
Sorry for the post review comment. This is a bit of a weird name though since time is always changing. Is this an availability change?